### PR TITLE
Update/meta descriptors

### PR DESCRIPTION
**Pull request by doingandlearning**
Adds Social Meta descriptors to each page.

### CODE REVIEW

**Comment by phivk** [comment]
@doingandlearning if I test a session page the meta description shows the general Gradual description:
https://www.heymeta.com/url/gradual-git-update-meta-descriptors-phivk.vercel.app/mozfest/sessions/how-to-foster-healthy-online-communities

I think the field on `$content` based `session` might be called different...

**Review by phivk — CHANGES_REQUESTED** [review]
if I test a session page the meta description shows the general Gradual description instead of the sessions 'summary' above the `<-- more -->` fold:
https://www.heymeta.com/url/gradual-git-update-meta-descriptors-phivk.vercel.app/mozfest/sessions/how-to-foster-healthy-online-communities

I think the field on `$content` based `session` might be called different...

**Comment by doingandlearning** [comment]
Ah -  I wasn't setting the top level description, just the socials. 

Ready for testing again @phivk

File: components/SocialHead.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import Logo from "~/assets/logo-gradual-white-text.svg"
+export default {
+  props: {
+    title: {
+      type: String,
+      required: true
+    },
+    description: {
+      type: String,
+      required: true
```
Comment (phivk) [inline components/SocialHead.vue:15]: why not a default value here?

File: components/SocialHead.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  <span v-if="false" />
+</template>
+
+<script>
+import Logo from "~/assets/logo-gradual-white-text.svg"
+export default {
+  props: {
+    title: {
+      type: String,
+      required: true
```
Comment (phivk) [inline components/SocialHead.vue:11]: why not a default value here?

File: pages/mozfest/index.vue [hunk 3]
```diff
@@ -1,5 +1,9 @@
 <template>
   <div>
+    <social-head
+     title="MozFest - Here to learn"
+     description=""
```
Comment (phivk) [inline pages/mozfest/index.vue:5]: leaving this empty results in empty twitter and og descriptions. Perhaps use a default value in SocialHead for the title and description props?

```
<meta data-n-head="ssr" data-hid="twitter:description" name="twitter:description" content="">
<meta data-n-head="ssr" data-hid="og:description" property="og:description" content="">
```
view-source:https://gradual-qpbwuzrfd-phivk.vercel.app/mozfest/

similar for other uses of SocialHead component

**Review by phivk — CHANGES_REQUESTED** [review]
see code comments.

Generally, I am not so sure about setting head tags in de body of a component. I suspect there is a more elegant JS based solution that doesn't touch component templates.
But let's leave that as a post MozFest exercise :)

**Comment by doingandlearning** [comment]
Good edge case discerning - I think this approach is probably best. Provide the defaults in nuxt.config.js and then overwrite if there is content to do so.

**Review by phivk — APPROVED** [review]

**Pull request merged**